dali/test/python/test_fw_iterators.py

```diff
@@ -428,6 +428,70 @@ def test_pytorch_iterator_not_fill_last_batch_pad_last_batch():
     assert len(next_img_ids_list_set) == data_size
     assert len(set(next_mirrored_data)) != 1
 
+class CustomPipe(Pipeline):
+    def __init__(self, batch_size, num_threads, device_id, num_gpus, data_paths):
+        super(CustomPipe, self).__init__(batch_size, num_threads, device_id,)
+        self.input = ops.FileReader(file_root = data_paths, shard_id = device_id, num_shards = num_gpus)
+        self.decode_gpu = ops.ImageDecoder(device = "mixed", output_type = types.RGB)
+        self.res = ops.RandomResizedCrop(device="gpu", size =(224, 224))
+
+        self.cmnp = ops.CropMirrorNormalize(device="gpu",
+                                            output_dtype=types.FLOAT,
+                                            output_layout=types.NCHW,
+                                            crop=(224, 224),
+                                            image_type=types.RGB,
+                                            mean=[0.485 * 255,0.456 * 255,0.406 * 255],
+                                            std=[0.229 * 255,0.224 * 255,0.225 * 255])
+
+    def define_graph(self):
+        jpegs, _ = self.input(name="Reader")
+        images = self.decode_gpu(jpegs)
+        images = self.res(images)
+        output = self.cmnp(images)
+        return output
+
+def test_pytorch_iterator_feed_ndarray():
```

Comment:
Is it possible to add PaddlePaddle test as well?

Reply:
will do